src/components/evaluation/EvaluationTable.vue

```diff
@@ -469,12 +493,30 @@ export default {
       })
     },
     updateEvaluationsSelected(evaluationId) {
+      const index = this.$_.indexOf(this.selectedEvals, evaluationId)
+      
+      if (index === -1) {
+        this.selectedEvals.push(evaluationId)
+      } else {
+        this.selectedEvals.splice(index, 1)
+      }
+
       this.toggleSelectEvaluation(evaluationId)
+
       this.$root.$emit('update-evaluations-selected')
     },
     selectInstructor(instructor) {
       instructor.emailAddress = instructor.email
       this.setPendingInstructor(instructor)
+    },
+    toggleSelectAll() {
+      if (this.allEvaluationsSelected) {
+        this.selectedEvals.map(id => this.updateEvaluationsSelected(id))
```

Comment:
@cesarvh It would be better to use `forEach()` or `this.$_.each()` instead of `map()` here, since `map()` returns a new array.